docsets/XMake/docset.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "XMake",
-    "version": "2.2.8",
+    "version": "2.2.9",
     "archive": "XMake.tgz",
```

Comment:
You need to update the archive at this path as well. `specific_versions` are completely separate from the main docset.